clippy_lints/src/methods/mod.rs

```diff
@@ -1892,7 +1892,9 @@ declare_clippy_lint! {
     /// trait.
     ///
     /// ### Why is this bad?
-    /// It is recommended style to use collect. See
+    /// If it's needed to create a collection from the contents of an iterator, the `Iterator::collect()`
```

Comment:
This should either be `Iterator::collect(self)`, `Iterator::collect(_)` or simply `_.collect()`.